ext/awscrt.stub.php

```diff
@@ -19,3 +19,33 @@ function aws_crt_event_loop_group_options_set_max_threads(object $elg_options, i
 function aws_crt_event_loop_group_new(object $options): object {}
 
 function aws_crt_event_loop_group_release(object $event_loop_group): void {}
+
+function aws_crt_credentials_options_new(): object {}
+
+function aws_crt_credentials_options_release(object $options): void {}
```

Comment:
We generally leave types out- object was added as an option in 7.2, so that will likely throw a warning or compiler error in earlier versions.

Reply:
This is the stubs file, it is never loaded at runtime, it is only used to generate the native argument info header. Types are required or it will fail to generate.

Reply:
Gotcha, thank you